src/main/java/ua/vedroid/cinema/controller/ShoppingCartController.java

```diff
@@ -0,0 +1,42 @@
+package ua.vedroid.cinema.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import ua.vedroid.cinema.model.dto.ShoppingCartDto;
+import ua.vedroid.cinema.service.MovieSessionService;
+import ua.vedroid.cinema.service.ShoppingCartService;
+import ua.vedroid.cinema.service.UserService;
+import ua.vedroid.cinema.service.mapper.ShoppingCartMapper;
+
+@RestController
+@RequestMapping("/shopping-carts")
+public class ShoppingCartController {
+    private final ShoppingCartService shoppingCartService;
+    private final MovieSessionService movieSessionService;
+    private final UserService userService;
+    private final ShoppingCartMapper mapper;
+
+    public ShoppingCartController(ShoppingCartService shoppingCartService,
```

Comment:
let's either use @Autowired annotation above the constructor everywhere - or not use it it all